pylti/tests/test_common.py

```diff
@@ -153,6 +157,10 @@ def test_post_response_invalid_xml(self):
                u'handler_noauth/grade_handler')
 
         def request_callback(request, cburi, headers):
```

Comment:
Replace the list of arguments with `**kwargs` and then use `kwargs['headers']` in the body. Then you will not need to the disable.